src/select/parts/styles.scss

```diff
@@ -36,6 +37,14 @@ $checkbox-size: awsui.$size-control;
 
 .trigger {
   @include styles.text-overflow-ellipsis;
+  @include focus-visible.when-visible {
```

Comment:
What do these styles do? Cannot see them used anywhere

Reply:
This adds a focus ring around select when you activate it through keyboard. Other inputs do this already, but this was missing from all the inputs based on select

Reply:
First of all, there is already an existing focus outline on select component. What in the existing styles was not working?

Second, the `.trigger` styles are defined here: https://github.com/cloudscape-design/components/blob/015b28e11a7bb424412e61305a0bb4b9b0a151da/src/select/parts/trigger.tsx#L57

It does not receive `focusVisible` attributes, so these CSS selector never matches

Reply:
It matches on ButtonTrigger

https://github.com/cloudscape-design/components/blob/015b28e11a7bb424412e61305a0bb4b9b0a151da/src/internal/components/button-trigger/index.tsx#L9

Reply:
That line generates the following css-selector in the end

```
.awsui_trigger_dwuol_1dfho_116[data-awsui-focus-visible=true]:not(#\9):focus::before
```

In order to work, the attribute has to be on that node directly, not anywhere else. Button trigger has is own scoped styles, so selectors from this file have no effect there either

Reply:
Addressed in https://github.com/cloudscape-design/components/pull/891